swatch-core/src/main/resources/tag_profile.yaml

```diff
@@ -263,6 +273,19 @@ tagMetrics:
       prometheusMetric: rhacs:rox_central_cluster_metrics_cpu_capacity:avg_over_time1h
       prometheusMetadataMetric: subscription_labels
 
+   #RHODS metrics
+  - tag: rhods
+    metricId: redhat.com:openshift_data_science:cpu_hours
+    rhmMetricId: redhat.com:openshift_data_science:cpu_hours
+    awsDimension: cpu_hour
+    uom: CORES
+    billingWindow: MONTHLY
+    queryKey: addonSamples
+    queryParams:
+      resourceName: rhods
```

Comment:
Given our target PromQL query is supposed to be:
```
cluster:usage:workload:capacity_physical_cpu_hours
            * on(_id) group_right
            min_over_time(ocm_subscription_resource{resource_type="addon",resource_name='addon-open-data-hub', billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}[1h])
```

Setting the `resourceName` to `rhods` does not produce the correct query and we will get no results.

`resourceName` should be: `addon-open-data-hub`

Reply:
Will refactor